index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>SALAMCIK</title>
+    <link href="oformlenie.css"
+    rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+    <div class="info">
+    <img src="Брат Чингиз.jpg" alt="ПУ ПУ ПУ" class="chingiz">
+        <div class="info-text">
+        <h4><a href="https://ru.wikipedia.org/wiki/%D0%A7%D0%B8%D0%BD%D0%B3%D0%B8%D1%81%D1%85%D0%B0%D0%BD" target="_blank">
+        Чингисхан</a>(настоящее имя — Тэмуджин, Темучин, Темучжин)</h4>
+        Основатель и первый великий хан Монгольской империи, объединивший разрозненные монгольские и тюркские племена.</li>
+         Чингисхан организовал завоевательные походы монголов в Китай, Среднюю Азию, Ближний Восток (в основном территория современного Ирана), на Кавказ и в Восточную Европу.
+        </div>
+  </div>
+<div class="card bg-yellow">
+<h2>Сражения</h2>
+<ol class="list">
+    <li><a href="https://ru.wikipedia.org/wiki/%D0%9C%D0%BE%D0%BD%D0%B3%D0%BE%D0%BB%D1%8C%D1%81%D0%BA%D0%BE%D0%B5_%D0%B7%D0%B0%D0%B2%D0%BE%D0%B5%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5_%D0%A1%D1%80%D0%B5%D0%B4%D0%BD%D0%B5%D0%B9_%D0%90%D0%B7%D0%B8%D0%B8"target="_blank">
+        1209 год</a> 
+        — Чингисхан вторгся на территорию уйгуров и поставил в зависимость
+         от себя правителя киданей из династии Западное Ляо. </li>
+    <li><a href="https://ru.wikipedia.org/wiki/%D0%9C%D0%BE%D0%BD%D0%B3%D0%BE%D0%BB%D1%8C%D1%81%D0%BA%D0%BE-%D1%86%D0%B7%D0%B8%D0%BD%D1%8C%D1%81%D0%BA%D0%B0%D1%8F_%D0%B2%D0%BE%D0%B9%D0%BD%D0%B0"target="_blank">1215 год</a>

```

Comment:
```suggestion
    <li><a href="https://ru.wikipedia.org/wiki/%D0%9C%D0%BE%D0%BD%D0%B3%D0%BE%D0%BB%D1%8C%D1%81%D0%BA%D0%BE-%D1%86%D0%B7%D0%B8%D0%BD%D1%8C%D1%81%D0%BA%D0%B0%D1%8F_%D0%B2%D0%BE%D0%B9%D0%BD%D0%B0" target="_blank">1215 год</a>

```

Проставляй пробелы между аттрибутами